kroxylicious-kms-provider-hashicorp-vault/src/test/java/io/kroxylicious/kms/provider/hashicorp/vault/CertificateGenerator.java

```diff
@@ -100,7 +100,10 @@ public static X509Certificate generateSelfSignedX509Certificate(KeyPair pair) {
             var validFrom = Date.from(now);
             var validTo = Date.from(now.plus(Duration.ofDays(9999)));
             var certBuilder = new X509v3CertificateBuilder(
-                    new X500Name("CN=issuer"),
+                    // currently it's important that issuer == subject because we use these certs on vault to authenticate the client
```

Comment:
missing punctuation.  I needed to read the comment a couple of times before i parsed it correctly.